CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Changelog
 All notable changes to this project will be documented in this file. If you make a notable change to the project, please add a line describing the change to the "unreleased" section. The maintainers will make an effort to keep the [Github Releases](https://github.com/NREL/OpenOA/releases) page up to date with this changelog. The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [UNRELEASED]
+- The package name is changed from `operational_analysis` to `openoa` to be more consistent with how we expect to import OpenOA!
```

Comment:
Should you add a new blank line after this?

Reply:
Just fixed this now, thanks for catching the error!